app/layout.tsx

```diff
@@ -29,7 +29,7 @@ export default function RootLayout({
       >
         <Provider>
           <Navbar />
-          <div className='mx-4 mt-16 lg:mx-10'>{children}</div>
+          <div className='mt-16'>{children}</div>
```

Comment:
wouldn't it be easier to calculate in the additional margin values you're passing to the pages inside the root layout ? this way you don't have to specify this in every page like you are currently doing in the homepage
```suggestion
          <div className='mt-32 lg:mt-40'>{children}</div>
```

Reply:
I've taken a look at the styles and made adjustments based on your comments. Should be better now. Thanks :)